includes/html/graphs/generic_v3_multiline.inc.php

```diff
@@ -89,6 +89,8 @@
 
     if ($i && ($dostack === 1)) {
         $stack = ':STACK';
+    } else {
```

Comment:
probably better to just init $stack = '' outside the loop.

Reply:
changed